usaspending_api/common/matview_manager.py

```diff
@@ -28,6 +29,14 @@
             "mv_contract_award_search",
             {
                 "model": mv.ContractAwardSearchMatview,
+                "json_filepath": str(JSON_DIR / "mv_covid_financial_account.json"),
```

Comment:
This doesn't look correct

Reply:
Oh geez...how did I manage that slip up. Will fix that